DevEdu.API/DevEdu.Business/Services/CourseService.cs

```diff
@@ -44,5 +66,13 @@ public List<CourseTopicDto> SelectAllTopicsByCourseId(int courseId)
             var list = _courseRepository.SelectAllTopicsByCourseId(courseId);
             return list;
         }
+
+        private CourseDto FillCourse(CourseDto course, int id)
```

Comment:
we don't need it as a separate method as it will be used only in GetFullCourseInfo method